sources/packages/web/src/assets/css/base.scss

```diff
@@ -102,3 +108,34 @@ ul.no-bullets {
   list-style-type: none;
   margin-bottom: 0.5rem;
 }
+
+/** muted heading **/
+.muted-heading-text {
+  font-style: $font-style-normal;
+  font-weight: $font-weight-bold;
+  font-size: $font-header-medium;
+  line-height:$line-height-header-md;
+  /* Black */
+  color: $brand-black;
+  opacity: 0.5;
+  margin: 0px;
+}
+
+
+/** large heading **/
+.heading-x-large {
```

Comment:
Is it too different from the category-header-large that we have for form.io?
If not, I believe that we should use the same styles, moving the form.io styles into the base.scss